Bottom: 6ff1e4fd93dd6e680dc642a708e4a3cdaf838d8b
Top:    7929c914d8ca327a1870b47c1531f9512f60ec0a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 23:56:12 +0000

Remove left over debug


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 7039457..55571cf 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1251,8 +1251,6 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
 #endif
                                 copper_appearance,
                                 NULL));
-
-      //break;
     }
